packages/launchpad/cypress/e2e/project-setup.cy.ts

```diff
@@ -386,14 +388,18 @@ describe('Launchpad: Setup Project', () => {
         cy.findByRole('button', { name: 'Next step' }).should('have.disabled')
 
         cy.contains('Pick a framework').click()
-        cy.findByRole('option', { name: 'Create React App' }).click()
-        cy.findByRole('button', { name: 'Bundler(dev server) Webpack' }).should('not.exist')
-        cy.findByRole('button', { name: 'Next step' }).should('not.have.disabled')
+        cy.findByRole('option', { name: 'React.js' }).click()
+        cy.findByRole('button', { name: 'Next step' }).should('have.disabled')
```

Comment:
I guess this works cause they're just chainer assertions, but it reads weird
```suggestion
        cy.findByRole('button', { name: 'Next step' }).should('be.disabled')
```